lib/spans/span-event.js

```diff
@@ -191,11 +191,13 @@ class HttpSpanEvent extends SpanEvent {
 
     if (attributes.url) {
       this.addAttribute('http.url', attributes.url)
+      this.addAttribute('server.address', new URL(attributes.url).hostname)
```

Comment:
`new URL` is quite expensive. What is the shape of `attributes`? Do we have access to the desired information without URL parsing?

Reply:
That's a good question. In tests, at this point in the code, agentAttributes looks like this: 
` {
      'request.parameters.foo': 'bar',
      url: 'https://example.com/',
      procedure: 'GET',
      'http.statusCode': 200,
      'http.statusText': 'OK',
      nr_exclusive_duration_millis: 20.080708
    }
`
...so unlike our db props, we don't have an already-parsed hostname there. Regex seems expensive also, though maybe not as much as URL.

Reply:
No, URLs are a very bad target for regular expressions. How does `attributes` get defined? Are we able to pass through all of the properties we want from there?

Reply:
I'm not particularly thrilled with this solution, but I've updated to extract the host from the segment name, since that's available when we create the span event.